### PR TITLE
(DOCS-13185): Documenting binarySize agg expression

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**

### CODE REVIEW

File: source/reference/operator/aggregation/binarySize.txt [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  .. code-block:: javascript
+     :copyable: false
+
+     { "_id" : 4, "name" : "concert.jpg", "imageSize" : 269 }
+
+.. seealso:: 
+
+   - :pipeline:`$project`
+   - :pipeline:`$sort`
+   - :pipeline:`$limit`
```
Comment (kay-kim) [inline source/reference/operator/aggregation/binarySize.txt]: Can we add reference to $strLenBytes  to $binarySize and $binarySize to $strLenBytes pages?

That way we can help people know there is another operator and not have to have people memorize what operators are available.

Comment (kay-kim) [reply source/reference/operator/aggregation/binarySize.txt]: Oh, and add to redirects and to the release notes/4.4.txt

**Comment by jeff-allen-mongo** [comment]
@kay-kim tweaks have been made

**Comment by kay-kim** [comment]
@jeff-allen-mongo  -- I don't see the release notes updated.
https://github.com/mongodb/docs/blob/master/source/release-notes/4.4.txt

**Comment by jeff-allen-mongo** [comment]
Apologies @kay-kim, forgot to `git add` that one. It is there now.

**Pull request merged**